azure-pipelines.yml

```
@@ -47,15 +47,14 @@ steps:
 
 - powershell: .\build\Install-WindowsSdkISO.ps1 18362
 
-- task: DotNetCoreCLI@2
-  inputs:
-    command: custom
-    custom: msbuild
-    arguments: /t:restore .\ColorCode.sln
-  displayName: NuGet restore
+- script: msbuild -p:Configuration=Release -r -t:pack .\ColorCode.sln
```

```suggestion
- script: msbuild -p:Configuration=Release  -p:GenerateLibraryLayout=true -p:PackageOutputPath=build/nupkg -r -t:pack .\ColorCode.sln
```

Rather than the copy files, we should just specify the properties that the cake file was doing below here. Think that should come to the same resolution with one less step.